x-pack/legacy/plugins/maps/public/layers/styles/color_utils.test.js

```diff
@@ -61,7 +61,7 @@ describe('getColorRampCenterColor', () => {
 
 describe('getColorRampStops', () => {
   it('Should create color stops for color ramp', () => {
-    expect(getOrdinalColorRampStops('Blues')).toEqual([
+    expect(getOrdinalColorRampStops('Blues', 0, 1)).toEqual([
```

Comment:
This test case can be removed. Its identical to the one added for a custom range.